webpack.config.js

```diff
@@ -0,0 +1,54 @@
+//@ts-check
+
+'use strict';
+
+const path = require('path');
+const webpack = require('webpack');
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+  target: 'node', // vscode extensions run in a Node.js-context 📖 -> https://webpack.js.org/configuration/node/
+
+  entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
+  output: {
+    // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'extension.js',
+    libraryTarget: 'commonjs2',
+    devtoolModuleFilenameTemplate: '../[resource-path]'
+  },
+  devtool: 'source-map',
+  externals: {
+    vscode: 'commonjs vscode',
+    bufferutil: 'commonjs bufferutil',
+    'spawn-sync': 'commonjs spawn-sync',
+    'utf-8-validate': 'commonjs utf-8-validate'
+  },
+  plugins: [
+      new webpack.IgnorePlugin(/^\.\/locale$/, /handlebars$/)
```

Comment:
I ended up with an ignore for `electron` too - not sure if that is useful / necessary or just vestigial from one of my many dead ends.